third_party/gpus/rocm/BUILD.tpl

```diff
@@ -0,0 +1,128 @@
+licenses(["restricted"])  # MPL2, portions GPL v3, LGPL v3, BSD-like
+
+package(default_visibility = ["//visibility:public"])
+
+config_setting(
+    name = "using_hipcc",
+    values = {
+        "define": "using_rocm_hipcc=true",
+    },
+)
+
+config_setting(
```

Comment:
I do not think we need these duplicated here. Is there a reason to not reuse them from //tensorflow/BUILD?

Reply:
@gunan configs here are implemented following `tensorflow/third_party/gpus/cuda/BUILD.tpl` where CUDA-specific configs are specified. These CUDA-specific configs aren't in `//tensorflow/BUILD` either.

Since the purpose of this PR is to introduce ROCm-specific build scripts without major refactoring to existing TensorFlow build infrastructure I'd like to propose keep this file as is. Thoughts?